### ISSUE TITLE
Accessibility finding: Page isn't findable except via direct link

### ISSUE THREAD

**Issue by coforma-jamie** [department-of-veterans-affairs/va.gov-team#131885]
### Checklist item

WEB-245	Pages can be found in multiple ways

### Severity level

3, Medium. Should be fixed in 1-3 sprints post-launch.

### Page(s)/url(s)

http://preview-prod.vfs.va.gov/preview?nodeId=85549 (must be on VA network to access) 

### Details

The page is not findable via any of [VA.gov](http://va.gov/)'s nav. I also couldn't find it via search. 

Joel Calumpong, my contact for this work, says: 

> 
> This comes from Joni Chan (part of the sitewide content and IA team, and I believe she worked w/ Mikki Northuis):
> 

> > As for the AccessVA page that you're building, it will live on the root level but won't be linked from the menus. People would get to it via direct link instead of navigating.

### Reproduction steps

(nothing to reproduce)

### Proposed solution or next steps

Keeping this page essentially hidden is intentional - VA is slowly moving away from the features of this page and wants to remove it completely when possible. But in the meantime, there aren't multiple means of finding this page.

I'm not sure what to do here, but wanted to report it.

**Comment by jasonday** [comment]
@coforma-jamie it seems like a valid issue, that may be a "won't fix" based upon your findings. Leave it to team to decide. Labels added.

**Comment by erinrwhite** [comment]
> [@erinrwhite](https://github.com/erinrwhite) is this an entry/IA issue too?

@jasonday yes and no 😂 Because it's intentional, and we have involvement from our IA stakeholder @mnorthuis on this decision, it's a feature, not a bug for this application.

**Comment by mnorthuis** [comment]
This page will not be part of primary nav.  All of Veteran/family tools available on this page are already part of the core VA.gov experience.  Some people may have the original page bookmarked and that will now redirect to this page.  Once that is place and the page is published, it will may surface organically in search for anyone specifically looking for it, but it is not intended to be a destination page for Veterans.  

cc @jonichan-ia
